compiler/rustc_error_codes/src/error_codes/E0791.md

```diff
@@ -0,0 +1,39 @@
+Static variables with the `#[linkage]` attribute must have one of the following
+types, which are equivalent to a nullable pointer in C:
```

Comment:
Maybe qualify that this restriction applies to statics within external blocks.

Reply:
Done